api/progession/progressionRouter.js

```diff
@@ -6,7 +6,7 @@ const {
   checkMenteeProgress,
   validateProgressId,
 } = require('./progressionMiddleware');
-const { validateUser } = require('../middleware/generalMiddleware');
+const { validateUser } = require('../middleware/validationMiddleware');
```

Comment:
This change was made to match the change in naming convention to the validationMiddleware.js file.